recipes-bsp/rpi-u-boot-scr/rpi-u-boot-scr.bb

```diff
@@ -9,9 +9,12 @@ INHIBIT_DEFAULT_DEPS = "1"
 
 SRC_URI = "file://boot.cmd.in"
 
+BOOT_DEVTYPE ?= "${@oe.utils.conditional("ENABLE_USBBOOT", "1", "usb", "mmc", d)}"
```

Comment:
Same here. Default to `mmc`. It would make things a bit cleaner and more flexible for the future.